style.css

```diff
@@ -199,6 +199,321 @@ body {
+}
+
+.second-page__quantity {
+  border: #000 solid 1.5px;
+  padding: 5px;
+}
+
+.second__minus {
+  padding: 10px;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.second__minus:hover {
+  background-color: #fae8e4;
+  border-radius: 20%;
+}
+
+.second__quantity__input {
+  text-align: center;
+  padding: 10px;
+  width: 15px;
+}
+
+.second__plus {
+  padding: 10px;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.second__plus:hover {
+  background-color: #fae8e4;
+  border-radius: 20%;
+}
+
+.second__add-to-cart {
+  font: 500 14px 'Montserrat-Medium', sans-serif;
+  padding: 11px 44px;
+  border: 1.5px solid #000;
+  background-color: #fff;
+  cursor: pointer;
+  width: 275px;
+}
+
+.second__add-to-cart:hover {
+  box-shadow: 0 0 0 4px #000;
+  text-decoration: underline;
+  transition: .3s ease;
+}
+
+.second__more-items {
+  font: 500 14px 'Montserrat-Medium', sans-serif;
+  color: #000;
+  cursor: pointer;
+  text-decoration: none;
+  letter-spacing: .1px;
+  border-bottom: 2px solid transparent;
+  padding-left: 5px;
+}
+
+.second__more-items:hover {
+  border-bottom: 2px solid #000;
+  transition: .3s ease;
+}
+
+.second__more-vector {
+  padding-left: 14px;
+  width: 48%;
+}
+
+/* Третья странца */
+
+.third-page {
+  background: linear-gradient(to right, #ebe3d8 422px, #000 422px, #000 502px, #ebe3d8 502px);
+  min-height: 981px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.third-page__img {
+  padding-left: 96px;
+}
+
+.third-page__img-clothes {
+  width: 109%;
+  padding-top: 5px;
+  box-shadow: -2px 32px 37px -20px rgba(0,0,0,0.2);
+}
+
+.third-page__wrap {
+  position: relative;
+  right: 32px;
+  top: 10px;
+}
+
+.third-page__article {
+  background-color: #fae8e4;
+  width: 408px;
+  padding: 37px 60px 43px 50px;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 45px;
+}
+
+.third-page__title {
+  font: 700 45px 'Montserrat-Bold', sans-serif;
+  letter-spacing: -.8px;
+  margin-bottom: 12px;
+}
+
+.third-page__subtitle {
+  font: 600 16px 'Montserrat-Bold', sans-serif;
+  letter-spacing: 2.3px;
+  margin-bottom: 36px;
+  opacity: .5;
+}
+
+.third-page__description {
+  font: 600 16px/23px 'Montserrat-Regular', sans-serif;
+  letter-spacing: .4px;
+  margin-bottom: 60px;
+  opacity: .3;
+}
+
+.third-page__options {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 7px;
+}
+
+.third-page__quantity {
+  border: #000 solid 1.5px;
+  padding: 5px;
+  width: 96px;
+}
+
+.third__minus {
+  padding: 10px;
+  background-color: #fae8e4;
+  cursor: pointer;
+}
+
+.third__minus:hover {
+  background-color: #ebe3d8;
+  border-radius: 20%;
+}
+
+.third__quantity__input {
+  text-align: center;
+  font-size: 15px;
+  padding: 10px;
+  width: 15px;
+  background-color: #fae8e4;
+}
+
+.third__plus {
+  padding: 10px;
+  background-color: #fae8e4;
+  cursor: pointer;
+}
+
+.third__plus:hover {
+  background-color: #EBE3D8;
```

Comment:
Codacy found an issue: [Expected "#EBE3D8" to be "#ebe3d8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)